modules/sw360/sw360-workflow/src/main/java/org/eclipse/sw360/antenna/sw360/SW360MetaDataUpdater.java

```diff
@@ -125,14 +126,18 @@ private boolean isLicenseInSW360(License license) {
      *                                 updated if it already exists
      * @return the updated or newly created release entity
      */
-    public SW360Release getOrCreateRelease(SW360Release sw360ReleaseFromArtifact, boolean updateExisting) {
+    public SW360Release getOrCreateRelease(SW360Release sw360ReleaseFromArtifact, boolean updateExisting, boolean artifactHasPrecedence) {
```

Comment:
The parameter name _artifactHasPrecedence_ (which is used in other places as well) refers to a rather technical and implementation-specific mechanism. Its purpose is to decide whether the data in SW360 should be updated or not - so better name it accordingly, e.g. _updateSW360_ or something like that.

Reply:
I went with a mixture and chose `overwriteSW360Data`.